staging/src/k8s.io/apiserver/pkg/authentication/token/cache/cache_simple.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cache
+
+import "sync"
+
+// simple key/value cache with read/write locking
+type simpleCache struct {
```

Comment:
It's not that important but we could add a TODO to switch this to the [new builtin `sync.Map` in Go 1.9]( https://tip.golang.org/doc/go1.9#sync-map).